README.md

```diff
@@ -4,22 +4,17 @@ Docker-based development environment manager
 
 
 # Installation
-Currently the recommended way to install dusty is to clone the dusty repository, create dusty binaries using our script, and place them in your `/usr/local/bin` folder.
-These commands should do that:
+Currently the recommended way to install dusty is to use our installation script, which will install the latest
+version of dusty's binaries, and set up a dustyd daemon plist:
 ```
-git clone https://github.com/gamechanger/dusty.git
-cd dusty
-./setup/create_binaries.sh
-cp dist/dusty /usr/local/bin/dusty
-cp dist/dustyd /usr/local/bin/dustyd
+bash `curl -L https://raw.githubusercontent.com/gamechanger/dusty/master/setup/install.sh`
```

Comment:
You're right currently we're only versioning binaries, not the install script or the plist, which are also curled.  We should probably version those